### ISSUE TITLE
GHC 8.0.1-rc3 build failure

### ISSUE THREAD

**Issue by deepfire** [goldfirere/th-desugar#39]
```
[2 of 8] Compiling Language.Haskell.TH.Desugar.Reify ( Language/Haskell/TH/Desugar/Reify.hs, dist/build/Language/Haskell/TH/Desugar/Reify.o )

Language/Haskell/TH/Desugar/Reify.hs:186:24: error:
    Not in scope: data constructor ‘FamilyD’
    Perhaps you meant ‘FamilyI’ (imported from Language.Haskell.TH.Syntax)
builder for ‘/nix/store/sgxj8s6nn8z9pjkzx3k1n1zb36h7x2ib-th-desugar-1.5.5.drv’ failed with exit code 1
```

**Comment by deepfire** [comment]
Same goes for `8.0.1-rc3`.

**Comment by christiaanb** [comment]
The master branch (unreleased version 1.6) builds just fine on ghc8-rc2 (and rc3). Indeed, the version currently released on hackage, 1.5.5, does not.

**Comment by deepfire** [comment]
Oh, thank you!
